spec/ProtectedFields.spec.js

```diff
@@ -138,4 +141,246 @@ describe('ProtectedFields', function() {
       expect(fetchedUser.has('favoriteColor')).toBeTruthy();
     });
   });
+
+  describe('using the pointer-permission variant', () => {
```

Comment:
All tests that you added here are regarding the `fetch` operation. Do you mind to add some additional tests for the `find` operation? I am particularly interested to test if the user will be able to access their, and only their, protected fields.

Reply:
New tests using the `find` operation have now been added by [this commit](https://github.com/parse-community/parse-server/pull/5951/commits/701c096c7f683035927ac24838ff1f3998eda5ed)